libs/core/src/lib/micro-process-flow/components/micro-process-flow/micro-process-flow.component.ts

```diff
@@ -0,0 +1,246 @@
+import {
+    AfterContentInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChildren,
+    ElementRef,
+    Input,
+    OnInit,
+    OnDestroy,
+    Optional,
+    QueryList,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { ContentDensityService, RtlService } from '@fundamental-ngx/core/utils';
+import { Subscription } from 'rxjs';
+import { startWith } from 'rxjs/operators';
+import { MicroProcessFlowItemComponent } from '../micro-process-flow-item/micro-process-flow-item.component';
+
+@Component({
+    selector: 'fd-micro-process-flow',
+    templateUrl: './micro-process-flow.component.html',
+    styleUrls: ['./micro-process-flow.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        class: 'fd-micro-process-flow',
+        '[class.fd-micro-process-flow--independent-steps]': 'independentSteps === true',
+        '[class.fd-micro-process-flow--compact]': 'compact === true'
+    }
+})
+export class MicroProcessFlowComponent implements OnInit, OnDestroy, AfterContentInit {
+
+    /** Should connector between items be hidden. */
+    @Input()
+    independentSteps = false;
+
+    /** Whether to apply compact mode to the micro process flow. */
+    @Input()
+    compact: boolean;
+
+    /** Pagination transition speed */
+    @Input()
+    transitionSpeed = 0.3;
+
+    /** Pagination transition effect */
+    @Input()
+    transitionTimingFunction = 'ease';
+
+    /** Micro process flow items. */
+    @ContentChildren(MicroProcessFlowItemComponent)
+    items: QueryList<MicroProcessFlowItemComponent>;
+
+    /** Previous items outside the viewport. */
+    _previousItemsCount = 0;
+
+    /** Next items outside the viewport. */
+    _nextItemsCount = 0;
```

Comment:
why are these prefixed by `_`? If they are private it should be specified. If they are hidden, you need to add it in the typedocs